### ISSUE TITLE
Add highway shields for Romania

### ISSUE THREAD

**Issue by ZeLonewolf** [ZeLonewolf/openstreetmap-americana#99]
Not all European shields.  Romania's look awesome!
![image](https://user-images.githubusercontent.com/281482/151209117-ca12ac0b-259a-4990-9cab-0935dd7b94fe.png)

_Originally posted by @zekefarwell in https://github.com/ZeLonewolf/openstreetmap-americana/issues/98#issuecomment-1022391986_

**Comment by Mashin6** [comment]
Here is the mention problem with the ref parsing. Ref for this route is `DN1` but shields show only number `1`. It's not terrible, but it will not look as pretty.
https://www.openstreetmap.org/relation/38256
https://de.wikipedia.org/wiki/Drum_na%C8%9Bional_1

**Comment by zekefarwell** [comment]
Sounds like the `ref` values are wrong.  `ref=DN1` should be just just `ref=1` for the same reason that US Route 1 has `ref=1` instead of `ref=US1`.

**Comment by Mashin6** [comment]
Right, `DN1` would be better as `name=` or `alt_ref=`.  There doesn't seem to have been a formal `route=road` relation proposal and every community came up with their own implementation (it's not the only place where I've seen this). Now this causes problems for global projects like this one. :( 
We can try to suggest a change to their refs, but I am somewhat skeptical that this will be successfull.

**Comment by zekefarwell** [comment]
I think we can just render the refs as they are.  We can share with the Romanian mapping community and if they care that it doesn't look right, they can discuss what to do about it.

**Comment by Mashin6** [comment]
(20c67cb) So the motorway shields are all good:
![image](https://user-images.githubusercontent.com/5265707/151679629-4a3e7f57-e97a-4e6c-850a-938c596df878.png)
But the trunk shields don't look good:
![image](https://user-images.githubusercontent.com/5265707/151679703-efa3f7cd-2f0d-4ecb-81b0-de2b07c0ff8c.png)

Stretching the shield makes it a bit better, but the 4 letter are IMO borderline readable.
![image](https://user-images.githubusercontent.com/5265707/151679872-301bdcc4-b5af-4bd4-abc2-fd89668c1553.png)
Longer refs are bad.
![image](https://user-images.githubusercontent.com/5265707/151679977-03979cdc-83f3-4962-ba2f-368b1d51a521.png)

**Comment by 1ec5** [comment]
Perhaps we should split this issue into motorways (which are ready and trivial to implement) and trunk roads (which still need a resolution on `ref` prefixes).
